photocolle-sdk/photocolle-sdk.framework/README.mkd

```diff
@@ -1,31 +0,0 @@
-## INSTALLATION
```

Comment:
I think `README.mkd` should not be removed because this file describes required frameworks and libraries. Users can not know what frameworks and libraries are required if this file is removed.

I think this `README.mkd` should be revised. What do you think?